ui/src/components/Dashboard/SidebarList.jsx

```diff
@@ -79,6 +79,10 @@ export default class SidebarList extends Component {
           Provisioners
         </SidebarListItem>
 
+        <SidebarListItem to="/worker-manager" icon={<MixcloudIcon />}>
+          Worker Manager
+        </SidebarListItem>
+
```

Comment:
We would probably just need to remove `process.env.SHOW_WORKER_MANAGER`

https://github.com/taskcluster/taskcluster/blob/d6fddea345e02aa6075dc8cc80d89ddf47c949ce/ui/src/components/Dashboard/SidebarList.jsx#L72-L76

Reply:
We will also need to remove `SHOW_WORKER_MANAGER` from `.neutrinorc.js`.

Reply:
Oh, I didn't know that was a thing!

Reply:
why to remove that variable? I don't want worker manager to be available in, well, not production, but the current public beta of the UI

Reply:
We're going to have people using it to set up workers in GCP, so it does need to be visible.